addons/mail/static/src/models/attachment_viewer.js

```diff
@@ -221,15 +240,14 @@ registerModel({
         angle: attr({
             default: 0,
         }),
-        attachment: one('Attachment', {
-            related: 'attachmentList.selectedAttachment',
+        attachment: one('AttachmentViewerItem', {
+            compute: '_computeAttachment',
         }),
         attachmentList: one('AttachmentList', {
             related: 'dialogOwner.attachmentListOwnerAsAttachmentView',
-            required: true,
         }),
-        attachments: many('Attachment', {
-            related: 'attachmentList.viewableAttachments',
+        attachments: many('AttachmentViewerItem', {
```

Comment:
`attachment` and `attachments` should be renamed according to the new model (`attachmentViewerItem`) to avoid confusion